### PR TITLE
Chinese Translation for 1.18.2

### PR DESCRIPTION
**Pull request by ChuijkYahus**

### CODE REVIEW

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/boundless.json
Comment (Cactusstudent) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/boundless.json]:
```suggestion
    "第2件盔甲则再额外提升5%",
```

**Comment by KnightMiner** [comment]
I should be done with any major changes that require new translations after that last commit, just have bug fixes and more minor features remaining before the release of the update.

Is this PR about ready to merge or would you prefer waiting until post update?

**Comment by Cactusstudent** [comment]
> Is this PR about ready to merge or would you prefer waiting until post update?

Oh, thank you very much. I have basically completed the conversion of the file path. If possible, please merge this PR this time (Because the old translation is outdated long time) but before that, we still have some details to discuss. I think we can get these done in one day. At last, please review this before merging, to make sure there isn't any technical problem.

**Comment by KnightMiner** [comment]
Yeah, I have been behind on handling translations for awhile. I plan on holding off on the update until I process at least this one as its been active and looks mostly complete.

Worst case if anything is not finished here when its time for the update you can open a new PR with the remaining work, as there will probably be another bug fix release in the future.

**Comment by Cactusstudent** [comment]
@KnightMiner We've done all these work, looks good to us, hope there has no big bugs or typos. You can merge it. Thanks a lot for reviewing this.

**Review by Cactusstudent — APPROVED** [review]
lgtm，hope this pr will work successfully

**Comment by Cactusstudent** [comment]
> > Is this PR about ready to merge or would you prefer waiting until post update?
> 
> Oh, thank you very much. I have basically completed the conversion of the file path. If possible, please merge this PR this time (Because the old translation is outdated long time) but before that, we still have some details to discuss. I think we can get these done in one day. At last, please review this before merging, to make sure there isn't any technical problem.

File: src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json
Comment (KnightMiner) [inline src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json:1]: Accidental renames of protection modifiers, it should be `_protection`, nor `projectile`. Name should match the name from `modifier_id`

Same for other protection types

Comment (Cactusstudent) [reply src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json:1]: oh, thx. It based on the en_us, so if this was wrongly named, maybe en_us also need a correction?

Comment (KnightMiner) [reply src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json:1]: Huh, apparently english is incorrect there. Did it correctly on every other language.

Comment (KnightMiner) [reply src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/defense/protection/tconstruct_blast_projectile.json:1]: Looking through the rest, everything seems to load fine, so I'll just merge this then rename the files after merge

**Pull request merged**
